members/forms/activity_signup_form.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, *args, **kwargs):
                         "Betaling",
                         Field("payment_option", aria_describedby="paymentHelp"),
                         HTML(
-                            '<span class="paymentHelp"><p>Vælg <b>ikke</b> "andet er aftalt", med mindre der er en klar aftale med den aktivitets ansvarlige, ellers vil tilmeldingen blive annulleret igen.{% if activity.will_reserve %} Denne betaling vil kun blive reserveret på dit kort. Vi hæver den først endeligt d. 1/1 det år aktiviteten starter for at sikre, at {{ person.name }} er meldt korrekt ind i foreningen i kalenderåret.{% endif %}</p></span>'
+                            '<span class="paymentHelp"><p>Vælg kun <i>"andet er aftalt"</i>, <u>hvis</u> der er en klar aftale med den aktivitets ansvarlige, ellers vil tilmeldingen blive annulleret igen.{% if activity.will_reserve %} Denne betaling vil kun blive reserveret på dit kort. Vi hæver den først endeligt d. 1/1 det år aktiviteten starter for at sikre, at {{ person.name }} er meldt korrekt ind i foreningen i kalenderåret.{% endif %}</p></span>'
```

Comment:
Som en note kommer vi nok til at afskaffe muligheden for at vælge andet er aftalt på et tidspunkt, og gøre det muligt for afdelingskaptajnen at vælge.